src/bitbutler.sh

```diff
@@ -698,6 +701,69 @@ JSON
   esac
 }
 
+function selfupdate() {
+  REPOSITORY_SLUG="yiisoft/yii2"
```

Comment:
Forgot some testing stuff, hu?

Reply:
oh noes! Must have slipped in with one of the last fixes.